### DISCUSSION TITLE
Change missing default scene component log from error to warning

### DISCUSSION THREAD

**Discussion by Mathiaszero** [FirstGearGames/FishNet#491] [Feature Request]
The following error pops up in the log when the dev places a game object containing the default scene component that is not under the network manager: “NetworkManager not found on [game object containing default scene component] or any parent objects. DefaultScene will not work.” 

I do not think this should be an error, but rather a warning for the player notifying that it is in a different location than the usual one.  The game should be able to run fine just as if the player plays the game as if the component was never added in the first place.

**Comment by FirstGearGames** [comment]
This error is intentional as the component not functioning could cause breaking behavior for the developer.

I do think it's worth adding an 'enabled' option that if not ticked will skip checks and simply stop the component from functioning.

**Comment by FirstGearGames** [comment]
Marking as added for the enabled toggle. Debug will remain an error.
